### DISCUSSION TITLE
Who to include for reviewing doc changes

### DISCUSSION THREAD

**Discussion by KristofKoch** [GenericMappingTools/gmt#7223] [Documentation development discussion]
After making some minor contributions to the docs I always wondered who to include as reviewers. *core*? *gmt-maintainers*? Any guidance from the regulars on best practices is appreciated.

**Comment by maxrjones** [comment]
> After making some minor contributions to the docs I always wondered who to include as reviewers. _core_? _gmt-maintainers_? Any guidance from the regulars on best practices is appreciated.



> After making some minor contributions to the docs I always wondered who to include as reviewers. _core_? _gmt-maintainers_? Any guidance from the regulars on best practices is appreciated.

Perhaps it would be best to create a new gmt-docs team of people who would like to review these types of pull requests. I could be on that team, though my time has been limited lately. @KristofKoch, @Esteban82, @anbj would you be interested in reviewing these PRs as part of a docs team?

**Reply by Esteban82** [reply]
Good idea Max. I am in.

**Reply by anbj** [reply]
I'll be happy to join this group. Documentation rocks.

**Reply by KristofKoch** [reply]
I'm also in.

**Comment by KristofKoch** [comment]
Any update on this?

**Reply by maxrjones** [reply]
Thanks for the prompt, just created the team and set permissions for docs-related repositories. let me know if you notice any that I've missed
